installation/mac/vs-template.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Project Template for VS
+page_title: Telerik Project Template for Visual Studio on macOS
+description: "Learn more about the Telerik extensions and project template for Visual Studio on macOS supported by Telerik UI for .NET MAUI."
+slug: mac-project-template
+position: 2
+previous_url: /installation/mac/project-template
+---
+
+# Telerik Project Template for VS on macOS
+
+The Telerik UI for .NET MAUI project template for Visual Studio (VS) on macOS aims to help developers when creating .NET MAUI applications with Telerik UI for .NET MAUI components.
+
+The project template for VS on macOS is an add-in that improves the getting started experience for Telerik developers and is pre-configured with all requirements to run the Telerik UI for .NET MAUI components.
+
+## Accessing the Project Template
+
+1. Install the add-in by using either of the following ways:
+
+  * If you have already installed Telerik UI for .NET MAUI, navigate to the `/[installation-path]/Telerik UI for .NET MAUI [version]/Extensions` folder. It contains the `telerik.ui.for.maui.mpack` file.
+
+  * Download the `telerik.ui.for.maui.mpack` file from your Telerik account. For details on how to do this, go to the topic about the [available Telerik UI for .NET MAUI product files]({%slug download-product-files %}).
```

Comment:
One more space character is required in order to correctly nest the bullet list.